dali/test/python/test_ops_internals.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
You need to rename it to `test_operator_*` to be picked by the wildcard in test script, or add it manually to test.sh in the appropriate TLX_ folder.

Reply:
Done